lib/zone.ts

```diff
@@ -666,7 +673,17 @@ const Zone: ZoneType = (function(global: any) {
     performance && performance['measure'] && performance['measure'](name, label);
   }
   mark('Zone');
-  const checkDuplicate = global[('__zone_symbol__forceDuplicateZoneCheck')] === true;
+
+  // Initialize before it's accessed below.
+  // __Zone_symbol_prefix global can be used to override the default zone
+  // symbol prefix with a custom one if needed.
+  const symbolPrefix = global['__Zone_symbol_prefix'] || '__zone_symbol__';
```

Comment:
Not a fan of this now becoming part of the public API which needs to be documented etc...

One way to solve this would be:
```suggestion
  const symbolPrefix = '__zone_symbol__' +  new Date().getTime() + '__;
```

But my guess is that this would be very breaking because even if it is private API people are probably relying on it.

Reply:
That's my concern as well. Having a unique symbol prefix is exactly what I need. The customizable symbol prefix is a way to do this w/o introducing a breaking change.